culture/engineering-principles.md

```diff
@@ -52,6 +52,27 @@ most of our projects don’t get large enough for their stewardship to become bu
   - [Open Source Ideology](https://ashfurrow.com/blog/open-source-ideology/)
   - [iOS at Scale: Artsy](https://www.objc.io/issues/22-scale/artsy/)
 
+### Psychological Safety
+
+At it's core, engineering is the practice of learning. To learn effectively and to be productive, engineers
```

Comment:
I really like the phrasing of "engineering is a practice of learning" and how the following sentence captures how we intend to use psychological safety as a principle to support this practice.